algorithm/threejs/trianglesToThreejsDatasets.js

```diff
@@ -0,0 +1,39 @@
+import { toPlane } from '@jsxcad/math-poly3';
+
+const intern = (map, point, next, update) => {
+  const key = JSON.stringify(point);
+  let index = map[key];
+  if (index !== undefined) {
+    return index;
+  }
+  map[key] = next;
+  update(point);
+  return next;
+};
+
+export const trianglesToThreejsDatasets = (options = {}, ...triangularGeometries) => {
```

Comment:
Awesome. This is exactly how I was thinking to do this

Reply:
Cool. :)